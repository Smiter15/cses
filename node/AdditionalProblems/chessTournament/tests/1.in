10
5 4 2 6 6 3 4 7 8 5
