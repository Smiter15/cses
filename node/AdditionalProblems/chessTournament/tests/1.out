25
9 8
9 5
9 4
9 10
9 1
9 7
9 2
9 6
8 5
8 4
8 10
8 1
8 7
8 2
5 4
5 10
5 1
5 7
4 10
4 6
4 3
2 1
2 10
7 6
3 1
